src/Solcore/Desugarer/EmitCore.hs

```diff
@@ -51,7 +49,7 @@ data EcState = EcState
 initEcState :: Bool -> TcEnv -> EcState
 initEcState debugp env = EcState
    { ecSubst = emptyVSubst
-   , ecDT = Map.empty
+   , ecDT = Map.fromList builtinDataInfo
```

Comment:
just for my understanding, what is an `ecDT`?

Reply:
`ec`is the "emit core" prefix, and `DT` stands for `DataTable`